src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1385,6 +1389,41 @@ namespace
         }
         else
         {
+            asr::AOVContainer aovs;
+
+            IRenderElementMgr* re_manager = GetCOREInterface()->GetCurRenderElementMgr();
+            re_manager->SetDisplayElements(false);
+            for (size_t i = 0, e = re_manager->NumRenderElements(); i < e; ++i)
```

Comment:
`NumRenderElements()` returns `int`; use `int` to iterate (and then you can remove a cast below).